static/css/main.css

```diff
@@ -532,7 +532,19 @@ p code {
 }
 
 p.edition {
+    /* styles that are common, like bold and italic, maybe font size */
     font-style: italic;
+}
+
+.post .matter p.edition {
+    /* styles specific to edition on posts list page */
+    text-align: left;
+    font-size: 10px;
```

Comment:
It looks a little to small for my taste:
![afol-small-editions](https://user-images.githubusercontent.com/12373754/90268373-33c08f00-de57-11ea-8caf-9ae37cb44689.png)

If you change it as below, it will be the same size as month and year on the left, which will look pretty nice:
```suggestion
    font-size: 0.8em;
```
![afol-improvement-proposal](https://user-images.githubusercontent.com/12373754/90268518-6cf8ff00-de57-11ea-8caf-c68e2c233327.png)